extensions/redis-client/deployment/src/main/java/io/quarkus/redis/client/deployment/RedisBuildTimeConfig.java

```diff
@@ -10,4 +15,43 @@
      */
     @ConfigItem(name = "health.enabled", defaultValue = "true")
     public boolean healthEnabled;
+
+    /**
+     * Default Dev services configuration.
+     */
+    @ConfigItem(name = ConfigItem.PARENT)
+    public DevServiceConfiguration defaultDevService;
+
+    /**
+     * Additional dev services configurations
+     */
+    @ConfigItem(name = ConfigItem.PARENT)
+    @ConfigDocMapKey("additional-redis-clients")
+    public Map<String, DevServiceConfiguration> additionalDevServices;
+
+    @ConfigGroup
+    public static class DevServiceConfiguration {
+        /**
+         * Configuration for DevServices
+         * <p>
+         * DevServices allows Quarkus to automatically start Redis in dev and test mode.
+         */
+        @ConfigItem
+        public DevServicesConfig devservices;
+
+        @Override
+        public boolean equals(Object o) {
```

Comment:
Why do you need that?

Reply:
Because of https://github.com/quarkusio/quarkus/pull/17070/files#diff-57cd460fe124deade3d5a65ef342d6d147ab56daf345359d634aba196a2b8580R58